workshops/templates/workshops/admin_dashboard.html

```diff
@@ -39,6 +40,8 @@
     <table class="table table-striped">
     {% for event in current_events %}
     <tr>
+      <td class="text-center" width="25px">{% if event.is_swc %}{% bootstrap_tag "SWC" %}{% endif %}</td>
+      <td class="text-center" width="25px">{% if event.is_dc %}{% bootstrap_tag "DC" %}{% endif %}</td>
```

Comment:
@pbanaszkiewicz , @maneesha wants to have the labels aligned.

I replaced hard-coding with looping. Any opinions?